app/javascript/src/utilities/json-utils.jsx

```diff
@@ -0,0 +1,20 @@
+// @flow
+
+// eslint-disable-next-line flowtype/no-weak-types
+const toJsonString = (val: Object): string => JSON.stringify(val, null, 2)
+
+const fromJsonString = (json: string) => JSON.parse(json)
+
+const safeFromJsonString = (json: string) => {
+  try {
+    return fromJsonString(json)
+  } catch (err) {
+    throw new Error(`That doesn't look like a valid json: ${json}`)
```

Comment:
The idea of this method was to rescue the error... what'd be the difference between JSON.parse then? or in this case fromJsonString!?